t/03-testapi.t

```diff
@@ -109,6 +109,17 @@ type_string 'hallo', wait_screen_change => 3, max_interval => 10;
 is_deeply($cmds, [{cmd => 'backend_type_string', max_interval => 10, text => 'hal'}, {cmd => 'backend_type_string', max_interval => 10, text => 'lo'},]);
 $cmds = [];
 
+subtest 'type_string with wait_still_screen' => sub {
+    my $wait_still_screen_called = 0;
+    my $module = new Test::MockModule('testapi');
```

Comment:
Use tools/tidy, looks like your code is failing https://travis-ci.org/os-autoinst/os-autoinst/builds/277241064#L1728